CMakeLists.txt

```diff
@@ -57,8 +57,28 @@ set(CMAKE_DISABLE_SOURCE_CHANGES ON)
 set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
 
 # verify that git submodules have been cloned
-include(cmake/verifysubmodules.cmake)
-verify_submodules("${CMAKE_CURRENT_LIST_DIR}/.gitmodules")
+if(NOT DEFINED VCPKG_INSTALLED_DIR)
+    include(cmake/verifysubmodules.cmake)
+    verify_submodules("${CMAKE_CURRENT_LIST_DIR}/.gitmodules")
+endif()
```

Comment:
nice!

Reply:
But doesn't that potentially backfire when using a combination of vcpkg and submodules?

Reply:
I don't care